icu4c/source/i18n/messageformat2_formatter.cpp

```diff
@@ -24,6 +24,18 @@ U_NAMESPACE_BEGIN
 
 namespace message2 {
 
+    namespace functions {
```

Comment:
Please do not indent the contents of a namespace -- here and elsewhere.

Reply:
Fixed in [38b9065](https://github.com/unicode-org/icu/pull/3418/commits/38b9065748d0f7092f571fc23e855c277a350b2f) -- I didn't un-indent the rest of this file (the contents of `namespace message2` outside of `namespace functions`) to avoid making the diff too big, but I can do that if you prefer.

Reply:
Reformatting is better in a separate PR.

FYI: On the way from "tech preview" to "draft" for 78, we all need to take another look at the API vs. spec and vs. C++ coding standards, and there may be more cleanup changes from that.